teku/src/main/java/tech/pegasys/teku/cli/BeaconNodeCommand.java

```diff
@@ -134,50 +133,65 @@ private static class ConfigFileCommand {
       scope = ScopeType.INHERIT)
   private File configFile;
 
-  @Mixin(name = "Network")
-  private Eth2NetworkOptions eth2NetworkOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Network%n", headingKey = "Network")
```

Comment:
Since we're not using resource bundles for translation we can just specify the `heading` and omit `headingKey` on all of these.